.codecov.yml

```diff
@@ -26,5 +26,4 @@ comment:
   require_changes: no
 
 ignore:
-  - "build"
```

Comment:
ignore build folder?

Reply:
Stop ignoring the build folder, which is where jacoco puts the code coverage data

Reply:
Why it was ignored before then? Now is removed to boost test coverage?

Reply:
I'm not sure, it might be just a codecov default. There was no build directory before so it didn't matter, but Gradle puts everything there so codecov needs to look there to find the coverage data

Reply:
👍